examples/step-79/step-79.cc

```diff
@@ -2420,9 +2420,9 @@ namespace SAND
         const double barrier_size_exponent   = 1.2;
 
         barrier_size =
-          std::max(std::min(barrier_size * barrier_size_multiplier,
-                            std::pow(barrier_size, barrier_size_exponent)),
-                   min_barrier_size);
+          std::clamp(barrier_size * barrier_size_multiplier, 
```

Comment:
```suggestion
          std::clamp(barrier_size * barrier_size_multiplier,
```
here and below you'll either need to delete this trailing whitespace manually or use `clang-format` (via `make indent`). Let us know if you need help setting up `clang-format`!

Reply:
Thanks for the help! I couldn't get the make indent to work, but I have been just using clang-format -i <file> to fix everything.